### PR TITLE
[FIX] web: compile a button with id

### PR DESCRIPTION
**Pull request by Polymorphe57**

### CODE REVIEW

File: addons/web/static/tests/views/form/form_view_tests.js [hunk 1]
```diff
@@ -10805,4 +10805,47 @@ QUnit.module("Views", (hooks) => {
...
+            res_model: "partner",
+            views: [[false, "form"]],
+            context: {
+                form_view_initial_mode: "readonly",
+            },
+        });
+
+        // mode is "edit" because target="new"
+        assert.containsOnce(target, ".o_form_view .o_form_editable");
+    });
```
Comment (aab-odoo) [inline addons/web/static/tests/views/form/form_view_tests.js:10850]: those are rather act_window tests

**Review by aab-odoo — APPROVED** [review]
Could you move the two tests of the second commit ? Besides that, it looks good :-)

File: addons/web/static/tests/views/form/form_view_tests.js (hunk 1 above)
Comment (Polymorphe57) [reply addons/web/static/tests/views/form/form_view_tests.js]: done

**Review by aab-odoo — APPROVED** [review]
robodoo rebase-ff r+

**Pull request closed without merging**